app/controllers/submissions_controller.rb

```diff
@@ -0,0 +1,41 @@
+class SubmissionsController < ApplicationController
+  def index
+    @submissions = Submission.where(params[:submission_key])
+  end
+
+  def create
+    @submission = Content.find(params[:content_id]).submissions.create(submission_params)
```

Comment:
If you are going to go for the new approach, then you don't need an endpoint for submissions for every single content. You should just have ONE endpoint to create ALL answers for ONE submission.